mne/dipole.py

```diff
@@ -987,8 +980,8 @@ def _fit_dipole_fixed(min_dist_to_inner_skull, B_orig, t, guess_rrs,
         warn('Zero field found for time %s' % t)
         return np.zeros(3), 0, np.zeros(3), 0, np.zeros(6)
     # Compute the dipole moment
-    Q, gof, residual = _fit_Q(guess_data, whitener, proj_op, B, B2, B_orig,
-                              rd=None, ori=ori)[:3]
+    Q, gof, residual_noproj = _fit_Q(guess_data, whitener, B, B2, B_orig,
```

Comment:
@larsoner you don't pass a parameter anymore but fit results stay the same. Can you explain how you made this happen? you pass data with proj on?